controllers/volumereplicationgroup_controller.go

```diff
@@ -1512,3 +1518,41 @@ func (r *VolumeReplicationGroupReconciler) addVolsyncOwnsAndWatches(ctrlBuilder
 
 	return ctrlBuilder
 }
+
+func (r *VolumeReplicationGroupReconciler) addKubeObjectsOwnsAndWatches(ctrlBuilder *builder.Builder) *builder.Builder {
+	r.Log.Info("Kube object protection enabled; watch kube objects requests")
+
+	// Find if velero CRDs are present in the cluster
+	veleroCRDs := []string{
+		"backups.velero.io",
+		"backupstoragelocations.velero.io",
+		"restores.velero.io",
+	}
+
+	foundCRDS := true
+
+	for _, crd := range veleroCRDs {
+		installedCRD := &apiextensionsv1.CustomResourceDefinition{}
+		if err := r.APIReader.Get(context.TODO(), types.NamespacedName{Name: crd}, installedCRD); err != nil {
+			r.Log.Info("Velero CRD not found", "CRD", crd, "error", err)
+
+			foundCRDS = false
+		}
+	}
+
+	if !foundCRDS {
+		r.Log.Info("Velero CRDs not found; Kubernetes object protection won't work unless Velero/OADP is installed")
```

Comment:
Need to unify with the reconcile error message.

Reply:
The reconcile error message is in addition to this log. I think this message can be a shorter one.

Reply:
Sure but we need to use the same terms and style in both logs. Currently we have:
- Velero/OADP
- velero/oadp

My suggestion velero/openshift-adp in both.

Reply:
I need to remove this error message as this gets logged for Managed App VRGs too.

Reply:
I verified, it only logs once when the operator starts. That is ok.